vars/rollingUpgradePipeline.groovy

```diff
@@ -100,24 +100,23 @@ def call(Map pipelineParams) {
                     }
                 }
                 steps {
-                    catchError(stageResult: "FAILURE") {
-                        timeout(time: 10, unit: 'MINUTES') {
-                            script {
-                                wrap([$class: 'BuildUser']) {
-                                    dir('scylla-cluster-tests') {
-                                        checkout scm
-                                        ArrayList base_versions_list = params.base_versions.contains('.') ? params.base_versions.split('\\,') : []
-                                        supportedVersions = supportedUpgradeFromVersions(
-                                            base_versions_list,
-                                            pipelineParams.linux_distro,
-                                            params.new_scylla_repo
-                                        )
-                                        (testDuration,
-                                         testRunTimeout,
-                                         runnerTimeout,
-                                         collectLogsTimeout,
-                                         resourceCleanupTimeout) = getJobTimeouts(params, builder.region)
-                                    }
+                    catchError(stageResult: "FAILURE") {timeout(time: 10, unit: 'MINUTES') {
+                        script {
+                            wrap([$class: 'BuildUser']) {
+                                dir('scylla-cluster-tests') {
+                                    checkout scm
+                                    ArrayList base_versions_list = params.base_versions.contains('.') ? params.base_versions.split('\\,') : []
+
+                                    supportedVersions = supportedUpgradeFromVersions(
+                                        base_versions_list,
+                                        pipelineParams.linux_distro,
+                                        params.new_scylla_repo
+                                    )
+                                    (testDuration,
+                                     testRunTimeout,
+                                     runnerTimeout,
+                                     collectLogsTimeout,
+                                     resourceCleanupTimeout) = getJobTimeouts(params, builder.region)}
```

Comment:
i'm under impression this was changed by mistake, as i don't see any change here other than indentation and new line format

Reply:
other than this change, the fix itself LGTM

Reply:
reverted that, it was an artifact of rebasing to master with PyCharn doing some unnecessary formatting here